tachyon/math/finite_fields/finite_field_traits.h

```diff
@@ -12,13 +12,15 @@ namespace tachyon::math {
 
 template <typename T>
 struct FiniteFieldTraits {
+  static constexpr bool kIsFiniteField = false;
```

Comment:
Do we have a case in which a field would not be a finite field?

Reply:
From the perspective of `Eigen::Matrix`, yes, because you can construct a matrix of int. What we want is to ensure that only our finite field can call the `Random()` function.